blazor/gantt-chart/accessibility.md

```diff
@@ -69,7 +69,7 @@ The Blazor Gantt component followed the [keyboard interaction](https://www.w3.or
 <kbd>Home</kbd> | <kbd>Fn + Left Arrow</kbd> | Moves the focus to the first cell of the focused row.
 <kbd>End</kbd> | <kbd>Fn + Right Arrow</kbd> | Moves the focus to the last cell of the focused row.
 <kbd>Ctrl + Home</kbd> | <kbd>Command + Fn + Left Arrow</kbd> | Moves the focus to the first Cell of the first row in the gantt chart.
-<kbd>Ctrl + End</kbd> | <kbd>Command +  Fn + Right Arrow</kbd> | Moves the focus to the last Cell of the last row in the gantt chart.
```

Comment:
table layout for mac keys are still misaligned
https://blazor.syncfusion.com/documentation/gantt-chart/accessibility#keyboard-navigation